vault/resource_kubernetes_auth_backend_config_test.go

```diff
@@ -342,8 +342,58 @@ func TestAccKubernetesAuthBackendConfig_fullUpdate(t *testing.T) {
 	})
 }
 
-func testAccKubernetesAuthBackendConfigConfig_basic(backend, jwt string) string {
-	return fmt.Sprintf(`
+func TestAccKubernetesAuthBackendConfig_localCA(t *testing.T) {
```

Comment:
This test fails when I run it with the error `one of pem_keys or kubernetes_ca_cert must be set`. Since we're passing `""` in `testAccKubernetesAuthBackendConfigConfig_basic`, looks like we're not setting either. I added the suggestion below to my file and the tests passed for me

Reply:
Resolving since this looks to be fixed in v1.9.3